src/main/java/task/Event.java

```diff
@@ -0,0 +1,48 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task{
+    private String time;
+    private LocalDate dateTime;
+    private boolean isParsedDate;
+
+    public Event(String line) throws EmptyStringException{
+        super();
+        if(line.isBlank()){
+            throw new EmptyStringException("Event cannot be empty.");
+        }
+        String[] command = line.split(" \\/at ");
```

Comment:
Since this is a String array representing "commands", perhaps the plural form of command could be used instead.